src/Views/AddDog/AddDog.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import UpdateDogFrom from '../../components/UpdateDogFrom/UpdateDogForm';
+import { addDogToDataBase } from '../../services/updateDog';
+import './AddDog.css';
+
+export default function AddDog() {
+  const [dog, setDog] = useState({});
```

Comment:
We should either add a loading state or set some default properties inside of the dog state to avoid our inputs going from uncontrolled to controlled